src/apis.cpp

```diff
@@ -203,25 +203,42 @@ int gds_prepare_send(struct gds_qp *qp, gds_send_wr *p_ewr,
 
 int gds_stream_queue_send(CUstream stream, struct gds_qp *qp, gds_send_wr *p_ewr, gds_send_wr **bad_ewr)
 {
-        int ret = 0, ret_roll = 0;
-	gds_send_request_t send_info;
+    int ret = 0, ret_roll = 0;
```

Comment:
indentation problem.
you should switch your editor to indent=8 spaces (no tabs)